### PR TITLE
adhoc fix for 6uwi for mmCIF

### PR DESCRIPTION
**Pull request by jamesmkrieger**
partially helps with #1790 

We still need a fix for MMTF

### CODE REVIEW

File: prody/proteins/ciffile.py [hunk 1]
```diff
@@ -432,7 +436,12 @@ def _parseMMCIFLines(atomgroup, lines, model, chain, subset,
 
         alt = line.split()[fields['label_alt_id']]
         if alt not in which_altlocs and which_altlocs != 'all':
-            continue
+            try:
+                alt = ascii_uppercase[int(alt)]
```
Comment (dkoes) [inline prody/proteins/ciffile.py:440]: Does this work? Wouldn't it make the altloc 'B' when the default is 'A'?

Comment (jamesmkrieger) [reply prody/proteins/ciffile.py:440]: Oh, right, true. I need to subtract 1.

**Comment by dkoes** [comment]
But isn't this going to produce warnings in normal cases - whenever you have a non-interger altloc that isn't requested?

**Comment by jamesmkrieger** [comment]
> Not sure this works? What I'd like to to with parseMMTF is revert the requested altloc from A to the first altloc in the file if no atoms are selected.

Good point. We still want to filter them. Thanks

**Comment by dkoes** [comment]
And what if a file has altloc 'A' and altloc '1'? Then this will merge them.

**Comment by jamesmkrieger** [comment]
ok, the altloc masking works for CIF too

It looks more complicated to set it for PDB where things seem to be assigned dynamically through the loop.

**Comment by jamesmkrieger** [comment]
It looks like this breaks the multi-model case

**Pull request merged**